src/data/constants.js

```diff
@@ -0,0 +1,70 @@
+import translationEn from "../i18n/en.json";
+import translationJa from "../i18n/ja.json";
+
+export const TIME_FORMAT = "MMMM d yyyy, HH:mm";
+export const COLOR_ACTIVE = "rgb(223,14,31)";
+export const COLOR_CONFIRMED = "rgb(244,67,54)";
+export const COLOR_RECOVERED = "rgb(25,118,210)";
+export const COLOR_DECEASED = "rgb(55,71,79)";
+export const COLOR_TESTED = "rgb(164,173,192)";
+export const COLOR_TESTED_DAILY = "rgb(209,214,223)";
+export const SUPPORTED_LANGS = ["en", "ja"];
```

Comment:
Only used in one place, also the order of the consts would be better if it was logical or alphabetical.

Reply:
My opinion is to store constants in a separate file, so even if it was used only once it'll be better to store it with other constants, otherwise it'll be double work every time when this value will be needed again

Reply:
If a constant is only used in one place, what would be the advantage of pulling it into a separate file? 

Keeping it in the same file can make the code more readable.